shippers/factory.py

```diff
@@ -20,16 +20,16 @@ class ShipperFactory:
     @staticmethod
     def create_from_output(output_type: str, output: Output) -> CommonShipper:
         if output_type == "elasticsearch":
-            elasticsearch_output = ElasticSearchOutput(output.type, output.kwargs)
+            assert isinstance(output, ElasticSearchOutput)
```

Comment:
we should avoid `assert`ing at runtime, let's raise an exception instead

Reply:
the assertion is for type hinting so I can access "output.elasticserach_output_params" later on

do you suggest to add an if statement on `isinstance` check and raise an exception?
fine for me as long as I can achieve what I wrote above